tests/test_eatwell101.py

```diff
@@ -0,0 +1,83 @@
+# mypy: allow-untyped-defs
+
+from recipe_scrapers.eatwell101 import EatWell101
+from tests import ScraperTest
+
+
+class TestEatWell101Scraper(ScraperTest):
+
+    scraper_class = EatWell101
+
+    def test_host(self):
+        self.assertEqual("eatwell101.com", self.harvester_class.host())
+
+    def test_author(self):
+        self.assertEqual("Christina Cherrier", self.harvester_class.author())
+
+    def test_title(self):
+        self.assertEqual(
+            "Creamy Spinach Turkey Meatballs", self.harvester_class.title()
+        )
+
+    def test_category(self):
+        self.assertEqual(
+            "Chicken, Cook, Cooking &amp; Meals, main dish recipes, Poultry recipes, ",
```

Comment:
Let's add an [`html.unescape`](https://docs.python.org/3/library/html.html#html.unescape) call on this value to resolve the ampersands?

Reply:
Fixed! Also made an update to the ingredients method to resolve a depreciation warning for using text instead of string in the search for the ingredients header.